domain-service/src/main/java/com/domainservice/common/configuration/feignclient/user/UserClient.java

```diff
@@ -1,16 +1,17 @@
 package com.domainservice.common.configuration.feignclient.user;
 
-import com.domainservice.domain.post.post.model.dto.UserView;
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 
+import com.domainservice.domain.post.post.model.dto.UserView;
+
 @FeignClient(
-        name = "account-service"
-        //fallbackFactory = UserClientFallbackFactory.class
+	name = "account-service"
+	//fallbackFactory = UserClientFallbackFactory.class
 )
```

Comment:
여기 url이랑 configuration 설정 해줘야 하지 않나요?? 저는 설정을 했었는데 안하신거 같아서 여쭤봅니다. 만약 설정 파일이 해당 브랜치에 있다면 수정해주시는것도 괜찮을것 같습니다.
```suggestion
@FeignClient(
	name = "account-service",
	url = "localhost:8080",
	configuration = {UserClientConfiguration.class}
)
```